pkg/scraper/script.go

```diff
@@ -88,7 +84,7 @@ func (s *scriptScraper) runScraperScript(ctx context.Context, inString string, o
 
 	go handleScraperStderr(s.config.Name, stderr)
 
-	logger.Debugf("Scraper script <%s> started", strings.Join(cmd.Args, " "))
+	logger.Infof("Scraper script <%s> started", strings.Join(cmd.Args, " "))
```

Comment:
I don't think this is necessary.

Reply:
Changing the level or logging that the scraper script is running? This was mostly for the support cases were we get "ran the scraper, nothing in the log" and I think it'd be helpful to at least see that Stash ran the scraper successfully even if it didn't produce correct output. We always ask users to turn their logging level to Debug and then there's usually some output 🙂